src/components/SignupForm.jsx

```diff
@@ -0,0 +1,112 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import SignupButton from './SubmitButton';
```

Comment:
All components should be organized and exported through one file, that way you'll use one line for the import.
Eg. `import { SignupButton, LoginButton, SignupInputForm } from './components';`
If you are unclear, ask for clarity when we have our sync today.

Reply:
I have all the files in the components folder to be exported through one file named "index.js"

Reply:
Implementing this feedback is making jest not to import this file correctly and causing the test to fail so I had to import each file individually.